sample_tpl/tpl_tests.txt

```diff
@@ -56,6 +56,7 @@ agg-vec-filter.tpl,true,10
 static-agg.tpl,true,49995000
 distinct-agg.tpl,true,45
 delete.tpl,true,0
+compaction-insert-into.tpl,true,11
```

Comment:
Is there any way to test if the tuple is inserted into the index successfully? either here or in the block_compactor test. I think select works even without index in our part. (correct me if that's incorrect)

Reply:
The test also checks index inserts and it returns an error in case it fails.